lang/fpc/Portfile

```diff
@@ -292,18 +298,19 @@ if {${subport} eq "${name}"} {
     } elseif {${os.platform} eq "darwin" \
               && ${os.major} <= 12} { # 10.8, Mountain Lion
         post-extract {
+            depends_extract port:xar
```

Comment:
@kamischi This cannot work correctly. If `xar` is not installed, then it won’t be called. Dependencies must be set outside of port stages.